doc/source/developer/debug.rst

```diff
@@ -14,32 +14,49 @@ So we build both MySQL and Mroonga for debugging in development.
 How to build MySQL for debugging
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-As you can see in `MySQL :: MySQL 5.5 Reference Manual :: 2.9.2 Installing MySQL from a Standard Source Distribution`_, you can build MySQL for debugging by passing ``-DWITH_DEBUG=yes`` option in CMAKE options.
+As you can see in `MySQL :: 2.9.4 Installing MySQL Using a Standard Source Distribution`_, you can build MySQL for debugging by passing ``-DWITH_DEBUG=yes`` option in CMAKE options.
 
 The procedure from download to build is the following. ::
 
   % mkdir -p ~/work/
   % cd ~/work/
-  % wget http://ftp.jaist.ac.jp/pub/mysql/Downloads/MySQL-5.5/mysql-5.5.13.tar.gz
-  % tar xvzf mysql-5.5.13.tar.gz
-  % cd mysql-5.5.13
-  % cmake . -DCMAKE_INSTALL_PREFIX=/tmp/local -DWITH_DEBUG=yes
+  % wget https://github.com/mysql/mysql-server/archive/refs/tags/mysql-8.0.29.tar.gz
+  % tar xvzf mysql-8.0.29.tar.gz
+  % mkdir mysql-server-mysql-8.0.29-build
+  % cd mysql-server-mysql-8.0.29-build
+  % cmake ../mysql-server-mysql-8.0.29 \
+      -DCMAKE_INSTALL_PREFIX=/tmp/local \
+      -DWITH_DEBUG=yes \
+      -DDOWNLOAD_BOOST=yes \
+      -DWITH_BOOST=../boost 
```

Comment:
We can use Boost bundled MySQL archive instead of downloading it at build time.

Reply:
I see.
Fixed it.